packages/plugins/RedPacket/src/SiteAdaptor/FireflyRedPacketAccountItem.tsx

```diff
@@ -6,9 +6,9 @@ import { Icons } from '@masknet/icons'
 import { EVMExplorerResolver } from '@masknet/web3-providers'
 import { openWindow } from '@masknet/shared-base-ui'
 
-const useStyles = makeStyles()((theme) => ({
+const useStyles = makeStyles<{ isDarkFont?: boolean }>()((theme, { isDarkFont }) => ({
     linkIcon: {
-        color: theme.palette.maskColor.secondaryMainDark,
+        color: isDarkFont ? theme.palette.maskColor.secondaryMainDark : theme.palette.maskColor.secondaryMain,
```

Comment:
`theme.palette.mode`

Reply:
Need this props to distinguish between two situations. One of the situation colors does not change with the theme.

Reply:
maybe you should wrap a dark theme provider outside of it?